src/webview/MessageList.js

```diff
@@ -290,6 +292,12 @@ class MessageList extends Component<Props> {
     return (
       <WebView
         useWebKit
+        {...Platform.select({
+          ios: {
```

Comment:
Ah, this is a neat way of expressing this conditional!

Reply:
I like it, if you do!

Maybe worth noting: It took some effort of reading the code for me to confirm that this way of selecting iOS doesn't contribute to unwanted rerenders of the WebView (which is not a PureComponent), which could be caused by new objects created and passed as props when MessageList's `render` is called. A new object _is_ created, but then that object is spread, to give the `startInLoadingState` and `this.getBlankWebView` props that don't change or get new references on MessageList's `render`.

The existing `source`, `originWhitelist`, and `ref` (there's a [new ref API](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs) that doesn't tempt the creation of a new callback on each render) would seem to suffer from this problem. In practice, MessageList's shouldComponentUpdate returning false probably prevents most or all of the re-renders, but it's [noted](https://reactjs.org/docs/react-component.html#shouldcomponentupdate) that this isn't a 100% effective way to prevent re-renders, and its behavior may change:

> In the future React may treat shouldComponentUpdate() as a hint rather than a strict directive, and returning false may still result in a re-rendering of the component.

It looks like there is an existing issue relating to re-renders of the WebView, but I'm not fully sure what's causing those re-renders specifically: https://github.com/zulip/zulip-mobile/issues/3026.